shell/platform/windows/text_input_plugin.cc

```diff
@@ -356,6 +416,40 @@ void TextInputPlugin::SendStateUpdate(const TextInputModel& model) {
   channel_->InvokeMethod(kUpdateEditingStateMethod, std::move(args));
 }
 
+void TextInputPlugin::SendStateUpdateWithDelta(const TextInputModel& model,
+                                               TextEditingDelta* delta) {
```

Comment:
It looks like we can pass this by const reference.

Reply:
Finally got my Windows machine setup and was able to make this change and verify that it works 👍